docs/install-alternatives.md

```diff
@@ -218,18 +286,12 @@ For an example, see this [answer on askubuntu][askubuntu].
 
 <!-- LINKS -->
 
+[uninstall]: /docs/uninstall-alternatives
```

Comment:
This link 404's

Reply:
it links to a page which was landing in another branch. I will refresh